src/storage/kv/rocksdb_engine.rs

```diff
@@ -72,6 +72,7 @@ impl Drop for RocksEngineCore {
     }
 }
 
+/// **Test functionality:** The RocksEngine is based on `RocksDB`.
```

Comment:
It's not obvious to me what `**Test functionality:**` means, and how the text after the colon relates to it. I would guess that it means this type is only for testing, though the name of the type doesn't indicate so.

Reply:
Clarified, thanks!